src/CredentialModel.cpp

```diff
@@ -307,15 +307,25 @@ qint8 CredentialModel::getAvailableFavorite(qint8 newFav)
     }
 }
 
-QModelIndex CredentialModel::getServiceIndexByName(const QString &sServiceName, int column) const
+QModelIndex CredentialModel::getServiceIndex(const QString &sServiceName, Qt::MatchFlag flag, int column) const
```

Comment:
is the matchflag compatible with old qt versions? :)

Reply:
Yes, Qt::MatchExactly value was already used from the enum earlier. (line 312)